jacorb-name-server/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM jboss/base:latest
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /opt/jboss
+EXPOSE 3528
+
+ENV JACORB_VERSION=2.3.1 \
+    JACORB_HOME=/opt/jboss/jacorb \
+    NAME_SERVER_PORT=3528 \
+    PATH=$PATH:/opt/jboss/jacorb/bin
+
+USER root
+
+RUN yum install -y ant \
+    && yum clean all
+
+USER jboss
+
+RUN curl -LOk http://www.jacorb.org/releases/$JACORB_VERSION/jacorb-$JACORB_VERSION-bin.zip \
```

Comment:
You could probably tag name server as 2.3.1 and jts as 5.2.1.Final

On Mon, Aug 24, 2015 at 10:23 AM, Marek Goldmann <notifications@github.com>
wrote:

> In jacorb-name-server/Dockerfile
> <https://github.com/jboss-dockerfiles/narayana/pull/2#discussion_r37733184>
> :
>
> > +WORKDIR /opt/jboss
> > +EXPOSE 3528
> > +
> > +ENV JACORB_VERSION=2.3.1 \
> > +    JACORB_HOME=/opt/jboss/jacorb \
> > +    NAME_SERVER_PORT=3528 \
> > +    PATH=$PATH:/opt/jboss/jacorb/bin
> > +
> > +USER root
> > +
> > +RUN yum install -y ant \
> > +    && yum clean all
> > +
> > +USER jboss
> > +
> > +RUN curl -LOk http://www.jacorb.org/releases/$JACORB_VERSION/jacorb-$JACORB_VERSION-bin.zip \
>
> Builds are scheduled. I have troubles with setting proper version tags to
> these images, since I have no idea what it could be right now. But latest
> will be available soon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jboss-dockerfiles/narayana/pull/2/files#r37733184>.
>